package.json

```diff
@@ -25,6 +25,7 @@
         "foundation-sites": "^6.4.3",
         "handlebars": "^4.7.6",
         "history": "^4.7.2",
+        "is-promise": "^4.0.0",
```

Comment:
accidentally added?

Reply:
Nope, intentional.  If we want to avoid the extra dependency, I'm happy to copy it rather than adding it as a dependency since it's only a couple of lines

Reply:
Can I ask we try and minimise the argo-ui dependencies only to things is needs? argo-ui dependencies are 80% of Snyk the security alerts we get - so we create work for ourselves.

Reply:
Fixed.  Once https://github.com/argoproj/argo-ui/pull/70 has been reviewed/merged, I'll go back and add some unit tests for the new `isPromise` function.